index.d.ts

```diff
@@ -16,7 +16,7 @@ interface EggRedisOptions {
 
 declare module 'egg' {
   interface Application {
-    redis: Redis;
+    redis: Redis | Map<string, Redis>;
```

Comment:
`app.redis` is not a instance of `Map`, it's instance of `Singleton`.

Reply:
![image](https://user-images.githubusercontent.com/344530/52856744-30641e80-3160-11e9-895b-2eaa57a2d595.png)

see https://github.com/eggjs/egg/blob/ef20ff75633b6e83b115d32af603d0f4f34cb1e1/lib/core/singleton.js

Reply:
yes, so I fix it now.

Reply:
please check again.